# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../ptlib/compile-fix.patch
# Copyright (C) 2009 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -Nur ptlib-2.4.4-orig/plugins/vidinput_v4l/vidinput_v4l.cxx ptlib-2.4.4/plugins/vidinput_v4l/vidinput_v4l.cxx
--- ptlib-2.4.4-orig/plugins/vidinput_v4l/vidinput_v4l.cxx	2009-01-23 18:26:26.000000000 +0000
+++ ptlib-2.4.4/plugins/vidinput_v4l/vidinput_v4l.cxx	2009-01-23 18:31:23.000000000 +0000
@@ -443,11 +443,11 @@
   return result;
 }
 
-PMutex creationMutex;
+PMutex _creationMutex_;
 static 
 V4LNames & GetNames()
 {
-  PWaitAndSignal m(creationMutex);
+  PWaitAndSignal m(_creationMutex_);
   static V4LNames names;
   names.Update();
   return names;
